tests/data_context/store/test_ge_cloud_store_backend.py

```diff
@@ -282,10 +282,10 @@ def test_list_keys(
 @pytest.mark.unit
 def test_remove_key(
     construct_ge_cloud_store_backend: Callable[
-        [GeCloudRESTResource], GeCloudStoreBackend
+        [GXCloudRESTResource], GeCloudStoreBackend
```

Comment:
can we also rename the GeCloudStoreBackend?

Reply:
that may break existing implementations due to our usage of `instantiate_class_from_config` - do we perhaps want to deprecate that name?